freeipa/src/main/java/com/sequenceiq/freeipa/keytab/v1/KerberosMgmtV1Service.java

```diff
@@ -0,0 +1,59 @@
+package com.sequenceiq.freeipa.keytab.v1;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.freeipa.api.v1.keytab.model.ServiceKeytabRequest;
+import com.sequenceiq.freeipa.api.v1.keytab.model.ServiceKeytabResponse;
+import com.sequenceiq.freeipa.client.FreeIpaClient;
+import com.sequenceiq.freeipa.service.FreeIpaClientFactory;
+import com.sequenceiq.freeipa.service.FreeIpaService;
+import com.sequenceiq.freeipa.service.stack.StackService;
+import com.sequenceiq.freeipa.util.CrnService;
+
+
+@Service
+public class KerberosMgmtV1Service {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(KerberosMgmtV1Service.class);
+
+    @Inject
+    private CrnService crnService;
+
+    @Inject
+    StackService stackService;
+
+    @Inject
+    FreeIpaService freeIpaService;
```

Comment:
missing private

Reply:
It is addressed in https://github.com/hortonworks/cloudbreak/pull/5441